### PR TITLE
Adding r-fairmodels from CRAN, recipe gen via conda_r_skeleton_helper

### PR DESCRIPTION
**Pull request by dillonroach**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by dillonroach** [comment]
@conda-forge/help-r ready for review!

File: recipes/r-fairmodels/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/fairmodels_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/fairmodels/fairmodels_{{ version }}.tar.gz
+  sha256: 97c39e00fe05402bc65b03d05f14698ed8101266e837f3534dac09d8cb17b373
+
+build:
+  skip: True  # [win]
```
Comment (cbrueffer) [inline recipes/r-fairmodels/meta.yaml:16]:
```suggestion
  noarch: generic
```

**Review by cbrueffer — CHANGES_REQUESTED** [review]
Please mark the recipe `noarch` as suggested. The win build may fail, but it doesn't matter as the packages from the Linux build will be used in the end.

**Comment by dillonroach** [comment]
Thanks for the suggestion - change committed!

**Review by cbrueffer — APPROVED** [review]

**Pull request merged**